common/test/acceptance/tests/lms/test_lms_course_home.py

```diff
@@ -75,6 +75,11 @@ def test_course_home(self):
         # Check that the tab lands on the course home page.
         self.assertTrue(self.course_home_page.is_browser_on_page())
 
+        # Check that a success message is shown when selecting a course goal
+        # TODO: LEARNER-2522: Assure the correct message shows up for a particular goal choice
+        self.course_home_page.select_course_goal()
+        self.assertTrue(self.course_home_page.is_course_goal_success_message_shown())
```

Comment:
Because the success-message css could be anything, maybe we should insert at line 80:
```
self.assertFalse(self.course_home_page.is_course_goal_success_message_shown())
```

Reply:
I'm going to leave it because it matches the current implementation and really should be a Truth test, the follow up story can make sure to update this to match the CSS